test/test_derivs.jl

```diff
@@ -8,38 +8,48 @@ using Transform
 import Synthetic
 import WCS
 
-import GSL.deriv_central
+import ForwardDiff
+
 
 println("Running derivative tests.")
 
+function do_deriv_test(f::Function, x::Float64, claimed_dx)
+    # TODO: really should call ForwardDiff.gradient just once, with a whole
+    # vector of x values, rather than once per parameter to test
+    fwd_deriv = ForwardDiff.gradient(f)([x,])[1]
+
+    info("got $fwd_deriv; expected $claimed_dx")
+
+    @test_approx_eq_eps significand(fwd_deriv) significand(claimed_dx) 1e-4
```

Comment:
Well nothing new is going wrong...with this PR the derivatives are checked as closely as before. And the changes are all to the test code. 

It might be worth exploring more why some derivatives don't match before 4 significant digits, but I don't see any red flags here. Numerical stability varies with the method of computing something, for example,

    @test_approx_eq (1e10 - 1e-6 - 1e10) 1e-6

fails on my machine.